scripts/lua/if_stats.lua

```diff
@@ -1355,8 +1359,9 @@ print[[
 <script>
 ]] print(jsFormCSRF('deleteNetworkForm', true)) print[[
 ]] print(jsFormCSRF('editNetworksForm', true)) print[[
-function doAddDeleteButton(td_idx, callback_str) {
-   $("td:nth-child("+td_idx+")", $(this)).html('<a href="javascript:void(0)" class="add-on" onclick="' + callback_str + '" role="button"><span class="label label-danger">Delete</span></a>');
+function doAddDeleteButton(td_idx, callback_str, label) {
```

Comment:
I use it as a reminder for callbacks, usually invoked after some user action. Is this bad?